src/enums.ts

```diff
@@ -1,14 +1,14 @@
 export enum CellNavigationMode {
-  NONE = 'none',
-  CHANGE_ROW = 'changeRow',
-  LOOP_OVER_ROW = 'loopOverRow'
+  none = 'none',
+  changeRow = 'changeRow',
+  loopOverRow = 'loopOverRow'
 }
 
 export enum UpdateActions {
```

Comment:
I wonder if we should use `type UpdateActions='COPY_PASTE' | 'CELL_DRAG' | 'COLUMN_FILL' | 'CELL_UPDATE'` instead of enum. We can change in a separate PR